### PR TITLE
Make do_tests print out the error message in case a test fails

### PR DESCRIPTION
**Pull request by jougs**
Debugging failing tests on Travis is a pain. This PR modifies `do_tests.sh` to print the script's output to the commandline in case an error occurs. I suggest @heplesser and @DimitriPlotnikov as reviewers.

### CODE REVIEW

File: testsuite/do_tests.sh.in [hunk 1]
```diff
@@ -299,27 +299,32 @@ run_test ()
 
         msg_dirty=${param_failure##* ${exit_code} }
         msg_clean=${msg_dirty%%,*}
+        msg_error="$( cat "${TEST_OUTFILE}" )"
         if test "${msg_dirty}" != "${param_failure}" ; then
             explanation="${msg_clean}"
         else
             explanation="Failed: unexpected exit code ${exit_code}"
-            msg_error="$( cat "${TEST_OUTFILE}" )"
+	    unexpected_exitcode=true
```
Comment (heplesser) [inline testsuite/do_tests.sh.in]: Looks as if indentation is off by one.

Comment (jougs) [inline testsuite/do_tests.sh.in]: Thanks for noticing. I used tabs instead of spaces. Fixed by ab14b92.

**Comment by heplesser** [comment]
@jougs It works! Once you remove the intentionally failing test again, I will approve.

**Comment by DimitriPlotnikov** [comment]
@jougs I have one clarification question: what is precisely the added value of the script compared to 'make installcheck'?

**Comment by jougs** [comment]
@DimitriPlotnikov: It's not the case that one provides a benefit over the other. The two are parts of the same thing. `do_tests.sh` is *run by* the `installcheck` rule in the `make` process to run all tests in a sensible order.

**Comment by DimitriPlotnikov** [comment]
@jougs OK:) 
Yes, also +1 from my side.

**Pull request merged**